gamestonk_terminal/portfolio/README.md

```diff
@@ -41,6 +42,20 @@ Your  login information should be stored as environment variables in [config fil
 * [allyhold](#allyhold)
   * Look at Ally Invest Holdings
 
+[DEGIRO](#DEGIRO)
+* [hold](#hold)
```

Comment:
I would add login here as well because login is a command

Reply:
Added